pages/community/index.tsx

```diff
@@ -13,8 +24,65 @@ function Index() {
                 If you can't find any, we encourage you to organize one yourself!
                 JetBrains is here to provide help and support.
             </CommunityBanner>
+
+            <CommunityKeepInTouch links={[
+                {
+                    icon: SlackIcon,
+                    title: 'Slack',
+                    description: 'Let’s chat with us in real time',
+                    link: 'https://kotlinlang.slack.com/'
```

Comment:
There is an invitation link near a link to Kotlin Slack(https://kotlinlang.org/community/).
Let's ask Denis to add it.